lib/shared/permissions.js

```diff
@@ -198,6 +198,13 @@ exports.elevateCommand = (command, options) => {
   // for now, we should make sure we double check if the error messages
   // have changed every time we upgrade `sudo-prompt`.
 
+  }).catch((error) => {
+    return _.includes(error.message, 'is not in the sudoers file');
+  }, () => {
+    throw errors.createUserError({
+      title: 'Your user doesn\'t have enough permissions to proceed',
+      description: 'This application requires sudo privileges to be able to write to drives'
```

Comment:
The language everywhere else in Etcher uses "drives" so I think we should also use "drives" here for consistency. If we decide to change terminology in future, we'll need to make that change in many places in Etcher.